RLBotPack/TensorBot/src/bot.cfg

```diff
@@ -0,0 +1,31 @@
+[Locations]
+# Path to loadout config. Can use relative path from here.
+looks_config = ./appearance.cfg
+
+# Path to python file. Can use relative path from here.
+python_file = ./bot.py
+
+# Name of the bot in-game
+name = TensorBot
+
+# The maximum number of ticks per second that your bot wishes to receive.
+maximum_tick_rate_preference = 120
+
```

Comment:
It would be great if you added a reference to your requirements file here, like this: https://github.com/RLBot/RLBotPack/blob/3e4c81f722e5c9028387545a75e6c7d96ab9fa82/RLBotPack/VirxEB/VirxEB.cfg#L6

It will cause the GUI to prompt people to install TF.

Hopefully it can install properly by just running `pip install -r requirements.txt`...